recipe/meta.yaml

```diff
@@ -19,14 +19,14 @@ build:
     - lib/R/lib/
     - lib/
 
-requirements:  # [not win32]
-  build:  # [not win32]
-    - r-base  # [not win32]
+requirements:
+  build:
+    - r-base
     - posix  # [win64]
     - m2w64-toolchain  # [win64]
     - gcc  # [not win]
-  run:  # [not win32]
-    - r-base  # [not win32]
```

Comment:
This was an old one :smile:

Reply:
Indeed, so much progress during this time :)